code/core/src/phone/java/com/adobe/marketing/mobile/services/ui/message/views/MessageFrame.kt

```diff
@@ -174,7 +179,17 @@ internal fun MessageFrame(
                         }
                     )
             ) {
-                MessageContent(Modifier.height(heightDp.value).width(widthDp.value), inAppMessageSettings, onCreated)
+                MessageContent(
+                    Modifier
+                        .height(heightDp.value)
+                        .width(widthDp.value),
+                    inAppMessageSettings,
+                    inAppMessageEventHandler,
+                    onHeightReceived = { height ->
+                        if (inAppMessageSettings.fitToContent == true) heightDp.value = height.dp
```

Comment:
Is this `height` received from the Javascript already adjusted for density and recalculated? I assume not, because the js layer will likely not have those details...